packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -637,9 +663,12 @@ export class ConnectionFlow {
     }
   }
 
+  /**
+   * Watch all jobs related to the current trigger
+   */
   watchTriggerJobs() {
-    // can happen when the current trigger comes from realtime
     if (this.trigger) {
+      // can happen when the current trigger comes from realtime
```

Comment:
You can say `When trigger comes from realtime, cozy-stack doesn't add the current_state to the object. So we need to request the stack to get it`